SPECS/infinipath-psm/infinipath-psm.spec

```diff
@@ -47,15 +47,15 @@ Development files for the %{name} library.
 
 %prep
 %setup -q -n psm-%{version}
-%patch1 -p1
-%patch2 -p1
-%patch3 -p1
-%patch4 -p1
-%patch5 -p1
-%patch6
-%patch7 -p1
-%patch8 -p1
-%patch9 -p1
+%patch 1 -p1
```

Comment:
I changed this after seeing this warning: `%patchN is deprecated (10 usages found), use %patch N (or %patch -P N)`